revng-test-configure

```diff
@@ -179,7 +179,7 @@ class Variables:
             elif type(self.variables[name]) is not list:
                 error(f"Key {name} is defined with different types")
 
-            self.variables[name] += value
+            self.variables[name] = value + self.variables[name]
```

Comment:
Why?

Reply:
This way the inherited tag's variables get added _after_ those of the tag it inherits from. And clang cares about order of stuff like `--target=X` if there's more than one specified.